scanner/metadata/metadata.go

```diff
@@ -172,11 +172,18 @@ func (t Tags) MbzAlbumComment() string {
 	return t.getFirstTagValue("musicbrainz_albumcomment", "musicbrainz album comment")
 }
 
-// ReplayGain Properties
+// Gain Properties
+type GainType int
 
-func (t Tags) RGAlbumGain() float64 { return t.getGainValue("replaygain_album_gain") }
+// Used by getGainValue to determine which gain tags to looks for
+const (
+	AlbumGain GainType = iota
+	TrackGain
+)
+
+func (t Tags) RGAlbumGain() float64 { return t.getGainValue(AlbumGain) }
```

Comment:
After the change above this would become:
```suggestion
func (t Tags) RGAlbumGain() float64 { return t.getGainValue("replaygain_album_gain", "r128_album_gain") }
```